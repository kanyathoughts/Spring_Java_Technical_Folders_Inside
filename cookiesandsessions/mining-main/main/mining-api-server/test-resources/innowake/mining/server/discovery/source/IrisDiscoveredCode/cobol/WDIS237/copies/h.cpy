         01 FIELD-H         PIC X(10).
