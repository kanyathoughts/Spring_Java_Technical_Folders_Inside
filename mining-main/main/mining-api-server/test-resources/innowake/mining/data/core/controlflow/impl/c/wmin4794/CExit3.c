#include <stdio.h>
#include <stdlib.h>

int main(void)
{
	int exit_code = 10;
	printf("Termination using _Exit");
	_Exit(exit_code);
}
