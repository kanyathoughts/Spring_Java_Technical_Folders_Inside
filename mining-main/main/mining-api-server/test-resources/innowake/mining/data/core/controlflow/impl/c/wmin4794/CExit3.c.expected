  #5 CFunctionDefinition, Function Name : main, FROM={TestEntryPoint}, TO={#19}
   #19 CExpressionStatement, FROM={#5}, TO={#24}
     #21 CFunctionReference
   #24 CLibraryFunction, FROM={#19}, TO={TestHaltPoint}
     #26 CFunctionReference
